assets/webpack.config.js

```diff
@@ -121,8 +125,23 @@ module.exports = (env, options) => ({
   },
   devtool: 'source-map',
   optimization: {
-    minimize: process.env.NODE_ENV == 'production',
+    minimize: options.mode == 'production',
```

Comment:
Because we weren't setting NODE_ENV in our build, we weren't actually minimizing our .js files, for example, right now, this isn't minimized:
https://tokamak.oli.cmu.edu/js/oli_multi_input_authoring-dd7471757314849a789170675c6adb4f.js?vsn=d

While I did set NODE_ENV in the build, I also moved this to use options.mode like was probably intended.